src/base/_themes.scss

```diff
@@ -42,11 +42,20 @@
   --theme-opacity-del: 1;
 }
 
+/**
+ * Alternate theme
```

Comment:
It might be helpful to add just a _bit_ more of a comment explaining how this compares to light and dark?